doc/user_guide/rendering.md

```diff
@@ -2,25 +2,25 @@
 
 ## Creating the Renderer With Default Settings
 
-To render an SQL statement, you need to create an instance of the SQl  Statement Renderer (for example `CreateTableRenderer`) using the `create()` method of
-the Renderer class.
+To render an SQL statement, you need to create an instance of the SQl Statement Renderer (for example `CreateTableRenderer`) using the `create()` method of
+the Renderer class:
```

Comment:
We usually don't use hard wraps in the docs, so if you see them you can fix them inlining

Reply:
Thanks! I was not sure, so I left them :)